go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/textileio/go-tableland
 
-go 1.18
+go 1.19
```

Comment:
This _might_ fix the issue with building the binaries, but I don't have a good way to test.